src/vs/base/common/htmlContent.ts

```diff
@@ -16,7 +16,6 @@ export class MarkdownString implements IMarkdownString {
 
 	value: string;
 	isTrusted?: boolean;
-	sanitize: boolean = true;
```

Comment:
Same, I don't understand this change.

Reply:
`sanitize` flag was a hack added before to render diagnostic message by generating and passing html through markdown string. Used this flag not to sanitize the html by the markdown renderer. With this change this flag is no longer needed and hence removed it.